active/tab/tabcontrol.md

```diff
@@ -0,0 +1,252 @@
+
+    // Remove tab from existing list
+    Tabs.Items.Remove(e.Tab);
+
+    // Add tab to list of Tabs on new page
+    newPage.AddItemToTabs(e.Tab);
+
+    // Set the Window's content to the new page
+    ElementCompositionPreview.SetAppWindowContent(newWindow, newPage);
+
+    // Show the window
+    await newWindow.TryShowAsync();
+}
+```
+
+## Databind to a set of tabs 
+
+``` xml
+<TabControl ItemsSource="{x:Bind TabItemCollection}" />
+```
+
+# Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples
+section.  For example, see the Remarks for the MediaPlayerElement 
+(https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MediaPlayerElement#remarks). -->
+
+## Keyboarding behavior
+### TAB and Arrow key behavior
+The TabControl has areas containing left content, right content, and the Tab content (containing 0 or more Tabs). When the user hits the TAB key, focus moves between the content areas. When focus moves into the Tab content area, the selected Tab gains focus. Once the user is in the Tab content area, pressing the arrow key will move focus between the tabs. Arrow focus is trapped inside the Tab content area.
+
+### Selecting a Tab
+When a Tab has focus, pressing SPACE or ENTER will select that Tab.
+
+### Shortcuts for moving between tabs
+Ctrl+TAB will select the next Tab.
+Ctrl+Shift+TAB will select the previous Tab.
+Tab selection cycles (meaning if the user has selected the last tab and presses Ctrl+TAB, the first tab becomes selected).
+
+Ctrl+1 through 8 will automatically select the corresponding Tab.
+Ctrl+9 will automatically select the last Tab.
+
+### Closing a Tab 
+Hitting CTRL+W or CTRL+F4 will automatically close the selected Tab (assuming it is closable). 
+
+### Keyboard guidance for App Developers
+The above sections outline built-in keyboarding behavior provided by the TabControl. However, there are certain expected keyboard shortcuts that you will be responsible for implementing. 
+* If your Tab control supports adding a new Tab, users expect CTRL+T to open a new tab
+* Consider maintaining a list of recently closed Tabs. If the user presses CTRL+SHIFT+T, they expect recently closed tabs to be reopened.
+* If users can perform more commands on a Tab than just closing the Tab (for example, pinning a Tab or duplicating a Tab), consider adding a context menu to the Tab. 
+
+# API Notes
+<!-- Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. -->
+
+### TabControl properties and events
+
+| Property | Type | Description |
+|:-------- |:---- |:----------- |
+| CanCloseTabs | bool | Default value for the item if it doesn't specify a IsClosable value. |
+| CloseButtonOverlay | enum | Describes the behavior of the close button of unselected tabs. Values are {Auto, OnHover, Persistent}. Default is Auto. In WinUI 2.2, Auto maps to Persistent.  |
```

Comment:
What is the reason for having this? I would suggest we just pick one behavior for now and implement only that.

Reply:
I agree that we should pick one behavior and only implement that. Preferably, the behavior is 'persistent' so it aligns with the new version of Edge using Chromium.

Reply:
> I agree that we should pick one behavior and only implement that. Preferably, the behavior is 'persistent' so it aligns with the new version of Edge using Chromium.

The Edgium tabs, obviously derive from the Chromium project, so it may be worth asking someone on that team to provide some guidance.

There are also the WPF Tabs, and the Win32 Tabs to derive from.

Reply:
What is the reason for the change from Edge UWP tabs (where the closeButtonOverlay is of type OnHover)? I'm just looking at Firefox, Edge Chromium,...and I have to say having a close button on *every* tab seems like overkill to me. In edge UWP, the additional space is used to display more of the tab title which seems far more useful to me then having a close button on every tab.

The only reason I could see right now is better touch support, i.e. you can directly close unselected tabs with touch without having to first select them so that the close button will appear. Not sure if that is a valid point or how Edge UWP is handling this case (no touch inout capable display here).

Reply:
> I agree that we should pick one behavior and only implement that. Preferably, the behavior is 'persistent' so it aligns with the new version of Edge using Chromium.

Thanks @cinnamon-msft! For v1 we'll focus on persistent tabs and leave the hover tabs for v2. 

There's still an open question about visualization, but we are actively iterating on visual design, so lets push pause on that topic until the team has a baseline point of view.